src/table/table.scss

```diff
@@ -122,265 +249,341 @@
+      }
     }
-    user-select: none;
 
-    &:hover {
-      @include themed {
-        background-color: t(iui-color-background-3) !important;
+    &.iui-negative {
+      &,
+      + .iui-expanded-content {
+        @include themed {
+          box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-negative);
+        }
+
+        *::selection {
+          @include themed {
+            background-color: rgba(
+              t(iui-color-foreground-negative-rgb),
+              t(iui-opacity-4)
+            );
+          }
+        }
       }
-    }
 
-    > .iui-content {
-      @include themed {
-        border-bottom: solid 1px t(iui-color-background-4);
+      .iui-status {
+        @include themed {
+          fill: t(iui-icons-color-negative);
+        }
       }
-      width: 100%;
-      display: flex;
-      justify-content: space-around;
-      align-items: center;
-      padding: $iui-table-density;
-      flex: 1;
+    }
 
-      > .iui-full-line {
-        flex: 1;
+    &.iui-expanded {
+      .iui-expander > .iui-icon {
+        transform: rotate(90deg);
       }
 
-      > .iui-controls {
+      + .iui-expanded-content {
         @include themed {
-          color: t(iui-text-color);
+          border-top: solid 1px t(iui-color-background-1);
         }
-        width: 300px;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        flex: 1;
+      }
+    }
+    // #endregion Selection
+  }
 
-        > a {
-          @include iui-anchor;
-          margin: 0 $iui-s;
-        }
+  // Empty & loading states
+  &.iui-message {
+    text-align: center;
+    height: 100%;
+    padding: $iui-xl;
+    @include themed {
+      color: t(iui-text-color-muted);
+      background-color: t(iui-color-background-2);
+    }
+  }
+}
 
-        > .iui-ellipsis {
-          margin: 0 $iui-s;
-        }
+@mixin iui-table-cell {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+  min-width: $iui-l * 6;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  overflow: hidden;
+  list-style: none;
+  flex-basis: $iui-l * 6;
+
+  > svg {
+    width: $iui-icons-default;
+    height: $iui-icons-default;
+    flex-shrink: 0;
+  }
 
-        > .iui-current,
-        > .iui-ellipsis {
-          @include themed {
-            color: t(iui-text-color);
-          }
-          cursor: default;
-        }
+  > .iui-status,
+  > .iui-progress-indicator-radial {
+    float: right;
+    margin-left: auto;
+  }
 
-        > .iui-current {
-          text-decoration: underline;
-          font-weight: $iui-font-weight-semibold;
-        }
+  > .iui-status {
+    margin-right: $iui-xs;
+  }
 
-        > a > .iui-icon {
-          @include themed {
-            color: t(iui-icons-color-actionable);
-            fill: t(iui-icons-color-actionable);
-          }
-          display: block;
-          width: $iui-icons-default;
-          height: $iui-icons-default;
+  > .iui-user-icon {
+    margin-right: $iui-s;
```

Comment:
This seems a bit random

Reply:
Removing this from the SCSS and placing it in the HTML's `<style>` in the next push.

Reply:
Fixed